articles/index.md

```diff
@@ -51,7 +53,7 @@ The current documentation includes the following topics.
 
 * [Microsoft Quantum Development kit release notes](xref:microsoft.quantum.relnotes).
 * [Quantum computing concepts](xref:microsoft.quantum.concepts.intro) includes topics such as the relevance of linear algebra to quantum computing, the nature and use of a qubit, how to read a quantum circuit, and more.
-* [Installation and validation](xref:microsoft.quantum.install) describes how to quickly set up your quantum development environment. Your Visual Studio environment will be enhanced with a compiler for the Q# language and templates for Q# projects and files.
+* [Getting Started with the Quantum Development Kit](xref:microsoft.quantum.install) describes how to quickly set up your quantum development environment from any platform (Windows, Linux, and Mac) using Visual Studio, Visual Studio Code or the command line.  
```

Comment:
I'd suggest emphasizing Windows 10 here, since we don't support 8 and earlier.

```suggestion
* [Getting Started with the Quantum Development Kit](xref:microsoft.quantum.install) describes how to quickly set up your quantum development environment from any platform (Windows 10, Linux, and Mac) using Visual Studio, Visual Studio Code or the command line.  
```